public/apps/configuration/utils/resource-utils.test.tsx

```diff
@@ -0,0 +1,36 @@
+/*
+ *   Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License").
+ *   You may not use this file except in compliance with the License.
+ *   A copy of the License is located at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file. This file is distributed
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
+ */
+
+import { generateResourceName } from './resource-utils';
```

Comment:
The utils pkg has 20+ files. It is a bit confusing to find a regular file since it is mixed with many test files. I would suggest to create a "test" sub pkg  under utils and move the test files over there for better readability.

Reply:
Good idea. Will do it in a separate PR.